client/src/components/Buttons/LinkButtonGreen/LinkButtonGreen.js

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+const LinkButtonGreen = ({ message, direction }) => {
```

Comment:
Do we really need to know that the Link is styled like a button? I would recommend to name this `LinkGreen` and the instead of `Button` in line 5 you could write `Wrapper`.